lib/registries/addon/my-registrations/styles.scss

```diff
@@ -51,6 +53,14 @@
             background-color: #e4e4e4;
         }
     }
+
+    .TabPane {
+        display: none;
+
+        &:global(.active) {
```

Comment:
Same here, but since it refers to a true global element in this context, it may be fine.